grpc.gemspec

```diff
@@ -43,7 +43,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec',              '~> 3.6'
   s.add_development_dependency 'rubocop',            '~> 1.41.0'
   s.add_development_dependency 'signet',             '~> 0.7'
-  s.add_development_dependency 'googleauth',         '>= 0.5.1', '< 0.10'
+  s.add_development_dependency 'googleauth',         '~> 1.0'
```

Comment:
this is actually generated from https://github.com/grpc/grpc/blob/133640507bc6191ee74bf76602b64ee5631c96b6/templates/grpc.gemspec.template#L48

Because we're just changing one line, you can just make the exact same change in that line of the template file -- can you please do that?